src/plugins/intel_cpu/src/output_mem_mgr.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "cpu_memory.h"
+#include "openvino/runtime/allocator.hpp"
+#include "memory_desc/cpu_memory_desc.h"
+
+namespace ov {
+namespace intel_cpu {
+
+class OutputAllocator : public Allocator {
```

Comment:
Since we are going to use `OutputMemoryMngr ` over `Allocator` we do not need this class at all.